imports/api/claims.js

```diff
@@ -0,0 +1,36 @@
+import { Meteor } from 'meteor/meteor'
+import { Mongo } from 'meteor/mongo'
+import { callAPI } from '../util/bugzilla-api'
+
+if (Meteor.isServer) {
+  /**
+   This publication is using the low-level meteor API to manage a published collection to the client
+   It has five available methods on 'this':
+   - added(collection-name, id, attributes) - adds a new document
+   - changed(collection-name, id, attributes) - changes an existing document and modifies the attributes
+   - removed(collection-name, id) - removed an existing document by id
+   - ready() - notifies the subscribed client of the initial success of the subscription with the initial data
+   - onStop(callback) - adds a handler for when a subscribed client removes its subscription
+   */
+  Meteor.publish('claims', function () {
+    if (!this.userId) return
+    const currUser = Meteor.users.findOne({_id: this.userId})
+    // Fetching simple data
+    callAPI('get', '/rest/bug/21', {token: currUser.bugzillaCreds.token})
```

Comment:
@nbiton Why `21`?

Reply:
This is just a dummy API request. It just so happened that id 21 was an existing claim on my own bugzilla instance...

Reply:
Ahh, thanks for explaining. How about using a placeholder variable that explicitly labels it as a dummy?